File: netCore/NEC.UnitTest/tests/NEC.Models.Tests/ConstructorTests.cs
using Xunit;

namespace NEC.Models.Tests
{
    public class ConstructorTests
    {

        [Fact]
        public void Car_NullMaker_ThrowsArgumentNullException()
        {
            // new Car(null, engine, seats);

            Assert.True(false);
        }

        [Fact]
        public void Car_EmptyMaker_ThrowsArgumentNullException()
        {
            // new Car("", engine, seats);

            Assert.True(false);
        }

        [Fact]
        public void Car_NullEngine_ThrowsArgumentNullException()
        {
            // new Car("notNullNorEmptyMaker", null, seats);

            Assert.True(false);
        }

        [Fact]
        public void Car_NullListOfSeats_ThrowsArgumentNullException()
        {
            // new Car("notNullNorEmptyMaker", engine, null);

            Assert.True(false);
        }

        [Fact]
        public void Car_EmptyListOfSeats_ThrowsArgumentNullException()
        {
            // new Car("notNullNorEmptyMaker", engine, emptyListOfSeats);

            Assert.True(false);
        }

        [Fact]
        public void Car_NotNullNorEmptyMakerAndNotNullEngineAndNotNullNorEmptyListOfSeats_ReturnsValidCar()
        {
            // new Car("notNullNorEmptyMaker", engine, seats);

            Assert.True(false);
        }
    }
}

File: java/nec.unit-test/src/test/java/nec/unit_test/ConstructorTests.java
package nec.unit_test;

import org.junit.Assert;
import org.junit.Test;

public class ConstructorTests {

    @Test
    public void car_nullMaker_throwsIllegalArgumentException() {
	// new Car(null, engine, seats);

	Assert.fail();
    }

    @Test
    public void car_emptyMaker_throwsIllegalArgumentException() {
	// new Car("", engine, seats);

	Assert.fail();
    }

    @Test
    public void car_nullEngine_throwsIllegalArgumentException() {
	// new Car("notNullNorEmptyMaker", null, seats);

	Assert.fail();
    }

    @Test
    public void car_nullListOfSeats_throwsIllegalArgumentException() {
	// new Car("notNullNorEmptyMaker", engine, null);

	Assert.fail();
    }

    @Test
    public void car_emptyListOfSeats_throwsIllegalArgumentException() {
	// new Car("notNullNorEmptyMaker", engine, emptyListOfSeats);

	Assert.fail();
    }

    @Test
    public void car_notNullNorEmptyMakerAndNotNullEngineAndNotNullNorEmptyListOfSeats_returnsValidCar() {
	// new Car("notNullNorEmptyMaker", engine, seats);

	Assert.fail();
    }
}
